content/en/docs/how-tos/adding-a-cluster-profile.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Add a New Cluster Profile"
+description: How to add a cluster profile to the test platform.
+---
+This document lays out the process of getting a new cluster profile added to the test platform so that jobs can begin using it.
+
+## What a Cluster Profile?
+
+The `cluster_profile` is a `ci-operator` concept that bundles together a couple of concepts to make it easier to configure jobs and steps that can operate on different cloud infrastructures.
+
+When a `cluster_profile` is added to a job or workflow, the following actions occur:
+
+ - all steps in the workflow will have [`credentials`](/docs/architecture/step-registry/#injecting-custom-credentials) mounted at `$CLUSTER_PROFILE_DIR` that contains credentials for cloud accounts, image registries, *etc*
+ - the test will implicitly ask for a [`lease`](/docs/architecture/step-registry/#implicit-lease-configuration-with-cluster_profile) and expose it with `$LEASED_RESOURCE`
+ - all steps in the test will implicitly declare [`dependencies`](/docs/architecture/ci-operator/#referring-to-images-in-tests) on imported OpenShift release images
+ - all steps will have a number of environment variables set, such a `$CLUSTER_TYPE`, `$IMAGE_FORMAT` and `$KUEBCONFIG`
+ 
+Generally, the major difference between `cluster_profile`s is the content of the credentials. These credentials are stored in the test platform clusters using naming convention: `cluster-secrets-<name>`; so, the `aws` profile stores credentials in `cluster-secrets-aws`.
+
+## Adding a New Cluster Profile
+
+When adding a new `cluster_profile`, three major steps must be taken: registering the profile inside of `ci-operator`, adding the new leases to `Boskos`, and providing the credentials.
```

Comment:
Technically it's not jsut ci-operator but also prowgen but I guess let's not complicate stuff